LICENSE.txt

```diff
@@ -9,8 +9,9 @@ are met:
 2. Redistributions in binary form must reproduce the above copyright
    notice, this list of conditions and the following disclaimer in the
    documentation and/or other materials provided with the distribution.
-3. The name of the author may not be used to endorse or promote products
-   derived from this software without specific prior written permission.
+3. Neither the name of the copyright holder nor the names of
+   its contributors may be used to endorse or promote products derived from
+   this software without specific prior written permission.
```

Comment:
There are few differences [The 3-Clause BSD License](https://opensource.org/licenses/BSD-3-Clause) 🤔 

https://github.com/scalaz/scalaz/commit/d87a28b3081b2ed69da5fb364d44fd6398fda9f6#diff-bf45f3e1f246049535fa2d23c91eb609R16